.drone/wait-for-instance.sh

```diff
@@ -3,17 +3,17 @@
 set -euxo pipefail
 
 getStatus() {
-    echo "$(curl -I -L -s -o /dev/null -w "%{http_code}" $1)"
+  curl -I -L -s -o /dev/null -w "%{http_code}" "${1}"
```

Comment:
Quoting the input prevents the script from breaking if ever the input contains spaces, line feeds, or glob characters.